commands/cli.go

```diff
@@ -420,31 +480,31 @@ func nukeDefaultSecurityGroups(c *cli.Context, regions []string) error {
 			logging.Logger.Errorf("[Failed] %s", err)
 		}
 	}
+
 	return nil
 }
 
-func confirmationPrompt(prompt string, maxPrompts int) (bool, error) {
+func confirmationPrompt(prompt string) (bool, error) {
 	ui.UrgentMessage("THE NEXT STEPS ARE DESTRUCTIVE AND COMPLETELY IRREVERSIBLE, PROCEED WITH CAUTION!!!")
 
-	shellOptions := shell.ShellOptions{Logger: logging.Logger}
-
-	// retry prompt on invalid input so user can avoid rescanning all resources
-	prompts := 0
-	for prompts < maxPrompts {
-		input, err := shell.PromptUserForInput(prompt, &shellOptions)
-		if err != nil {
-			return false, errors.WithStackTrace(err)
-		}
-
-		if strings.ToLower(input) == "nuke" {
-			return true, nil
-		}
+	confirmPrompt := pterm.DefaultInteractiveTextInput.WithMultiLine(false)
+	pterm.Println()
+	rawResp, err := confirmPrompt.Show(prompt)
+	if err != nil {
+		logging.Logger.Errorf("[Failed to render prompt] %s", err)
+	}
+	var userConfirmedNuke = false
+	response := strings.ToLower(strings.TrimSpace(rawResp))
+	if response == "nuke" {
+		userConfirmedNuke = true
+	}
 
-		fmt.Printf("Invalid value '%s' was entered.\n", input)
-		prompts++
+	if !userConfirmedNuke {
+		pterm.Println()
+		logging.Logger.Infof("You did not enter '%s', so exiting without nuking anything.", "nuke")
```

Comment:
Was the retry logic removed intentionally? sometimes the scans take a while to run and it'd be easy to tyep nuek instaed of nuke

Reply:
Aha - good catch, thanks! I think I misunderstood what it was doing and why. I'll add that logic back in.

Reply:
Replaced in 81a9054